forch/forchestrator.py

```diff
@@ -436,9 +445,43 @@ def _restore_states(self):
     def _restore_faucet_config(self, timestamp, config_hash):
         config_info, faucet_dps, _ = self._get_faucet_config()
         self._update_config_warning_varz()
-        assert config_hash == config_info['hashes'], 'config hash info does not match'
+
+        if config_hash == config_info['hashes']:
+            self._attempt_cancel_config_hash_clash_timer()
+        else:
+            LOGGER.warning('Config hash does not match')
+            self._attempt_start_config_hash_clash_timer()
+
         self._faucet_collector.process_dataplane_config_change(timestamp, faucet_dps)
 
+    def _attempt_start_config_hash_clash_timer(self):
```

Comment:
this shouldn't be "start a timer" -- but rather just a mark of when the last config hash clash was